website/docs/advanced/feature-toggle-variants.md

```diff
@@ -2,24 +2,100 @@
 id: toggle_variants
 title: Feature Toggle Variants
 ---
+<div class="alert alert--info" role="alert">
+  <em>Feature toggle variants</em> were introduced in <em>Unleash v3.2.0</em>.
+</div>
+<br/>
 
-> This feature was introduced in _Unleash v3.2.0_.
+Every toggle in Unleash can have something called _variants_. Where _feature toggles_ allow you to decide which users get access to a feature, _toggle variants_ allow you to further split those users into segments. Say, for instance, that you're testing out a new feature, such as an alternate sign-up form. The feature toggle would expose the form only to a select group of users. The variants could decide whether the user sees a blue or a green submit button on that form.
 
-Do you want to facilitate more advanced experimentations? Do you want to use Unleash to handle your A/B experiments? Say hello to feature toggle variants!
+Variants facilitate A/B testing and experimentation by letting you create controlled and measurable experiments. Check [our blog post on using Unleash for A/B/n experiments](https://www.getunleash.io/blog/a-b-n-experiments-in-3-simple-steps) for some more insights into how you can set it up.
 
-You can now extend feature toggles with multiple variants. This feature enables you to extend a feature toggle to divide your traffic among a set of variants.
+## What are variants?
 
-![toggle_variants](/img/variants.png 'Feature Toggle Variants')
+Whenever you create a feature toggle, you can assign it any number of _variants_. This is commonly done in cases where you want to serve your users different versions of a feature to see which performs better.
+
+A variant has four components that define it:
+- a **name**:
+
+    This must be unique among the toggle's variants. When working with a toggle with variants in a client, you will typically use variant name to find out which variant it is.
```

Comment:
Bit torn on this one. "use variant name"/"use the variant name"/"use the variant's name"?

```suggestion
    This must be unique among the toggle's variants. When working with a toggle with variants in a client, you will typically use the variant's name to find out which variant it is.
```